lib/src/json_schema/validator.dart

```diff
@@ -661,6 +718,40 @@ class Validator {
     return false;
   }
 
+  //////
+  // Helper functions to deal with unevaluatedItems.
+  //////
+  _pushUnevaluatedItemsContext() {
```

Comment:
Why **un**evaluated in some names here?

Reply:
I'm open to some better wording.
We are tracking the number of `evaluatedItems` for the `unevaluatedItemsContext`.